### PR TITLE
Port Exchange tests to support NanoSP

### PR DESCRIPTION
**Pull request by fbeutin-ledger**
- Modify test architecture to add automatic NanoSP testing
- Add binary files of exchanged apps compiled for NanoSP
- Add screenshots for NanoSP tests
- Split CI tests in 3 jobs to allow parallel testing and more accurate statuses

### CODE REVIEW

**Comment by fbeutin-ledger** [comment]
- Modify test architecture to add automatic NanoSP testing
- Add binary files of exchanged apps compiled for NanoSP
- Add screenshots for NanoSP tests
- Split CI tests in 3 jobs to allow parallel testing and more accurate statuses

File: test/src/test.fixture.js [hunk 1]
```diff
@@ -61,6 +64,7 @@ class ExchangeDeviceModel extends DeviceModel {
 const nano_environments: ExchangeDeviceModel[] = [
     { name: 'nanos', letter: 'S', path: NANOS_ELF_PATH, libs: get_applications('nanos'), sdk: '2.1'},
     { name: 'nanox', letter: 'X', path: NANOX_ELF_PATH, libs: get_applications('nanox'), sdk: '2.0.2'},
+    { name: 'nanosp', letter: "SP", path: NANOSP_ELF_PATH, libs: get_applications('nanosp')},
```
Comment (lpascal-ledger) [inline test/src/test.fixture.js:67]: If the `sdk` field is never used, it should be cleaned in the other environment, else it should be explicited (currently 1.0.2)

**Review by lpascal-ledger — APPROVED** [review]
Small comments, could be merged then

File: test/src/test.fixture.js (hunk 1 above)
Comment (fbeutin-ledger) [reply test/src/test.fixture.js:67]: The thing is that I don't actually know which version of Speculos Zemu currently uses.
I tried several options that do NOT work : "1.0.2", "1.0.0", "1"
Everytime Speculos returns an error "invalid SDK version"

Comment (fbeutin-ledger) [reply test/src/test.fixture.js:67]: Ok sdk version "1.0" did work so I'll use that

**Pull request merged**